### DISCUSSION TITLE
Extended versions with SMA for torrent downloaders, Qbittorrent etc.

### DISCUSSION THREAD

**Discussion by mrmuiz** [RandomNinjaAtk/docker-sabnzbd-extended#18] [Ideas]
I  realy love the SABNZB Extended version with SMA.
This would be nice if this existed for torrent downloaders like Qbittorrent.

**Comment by RandomNinjaAtk** [comment]
It would be neat to see something like that for those apps, but I do not use those apps at all, so it's not something I would create.
